packages/@aws-cdk/aws-lambda/test/integ.lambda.docker-arm64.ts

```diff
@@ -0,0 +1,23 @@
+import * as path from 'path';
+import { DockerPlatform } from '@aws-cdk/aws-ecr-assets';
+import { App, Stack } from '@aws-cdk/core';
+import { Architecture, DockerImageCode, DockerImageFunction } from '../lib';
+
+class TestStack extends Stack {
+  constructor(scope: App, id: string) {
+    super(scope, id);
+
+    new DockerImageFunction(this, 'MyLambda', {
+      code: DockerImageCode.fromImageAsset(path.join(__dirname, 'docker-arm64-handler'), {
+        platform: DockerPlatform.ARM_64,
```

Comment:
I wonder if there's a way to automatically set the `platform` based on the value of `architectures`.

Maybe pass `props.architectures` to `_bind()` here?

https://github.com/aws/aws-cdk/blob/1deea9005656c2f0f25c56e773145b6e0ebcbb1b/packages/%40aws-cdk/aws-lambda/lib/image-function.ts#L60-L69

then you can set the `platform` in `fromImageAsset`?